tests/src/application/validation/validators/email-validation.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { InvalidParamError } from '@/application/errors'
+import { EmailValidation, type EmailValidator } from '@/application/validation'
+
+const makeEmailValidatorStub = (): EmailValidator => {
+  class EmailValidatorStub implements EmailValidator {
+    isValid (email: string): boolean {
+      return true
+    }
+  }
+  return new EmailValidatorStub()
+}
+
+interface SutTypes {
+  sut: EmailValidation
+  emailValidatorStub: EmailValidator
+}
+
+const makeSut = (): SutTypes => {
+  const fakeFieldName: string = 'fieldName'
+  const emailValidatorStub = makeEmailValidatorStub()
+  const sut = new EmailValidation(fakeFieldName, emailValidatorStub)
+  return {
+    sut,
+    emailValidatorStub
+  }
+}
+
+describe('EmailValidation', () => {
+  it('should returns InvalidParamError if fieldName is not a string', () => {
+    const { sut } = makeSut()
+
+    const result = sut.validate({ fieldName: 11 })
+
+    expect(result).toStrictEqual(new InvalidParamError('fieldName'))
+  })
+
+  it('should returns InvalidParamError if fieldName is not a valid email', () => {
+    const { sut, emailValidatorStub } = makeSut()
+    jest.spyOn(emailValidatorStub, 'isValid').mockReturnValueOnce(false)
+
+    const result = sut.validate({ fieldName: 'invalid_email@mail.com' })
```

Comment:
Same here. Maybe you want avoid hardcoded strings and use variables or values with better descriptions